corona-warn-app/cwa-app-android
# Notification trigger adjustments (EXPOSUREAPP-12484)

This PR addresses [EXPOSUREAPP-12484](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12484)
Trigger behaviour for notfications has been optimizied.

How to test:
- scan invalid, blocked, expired certificates
- check the notifications (see ticket for details and cases)
